src/java.base/share/classes/java/lang/String.java

```diff
@@ -250,7 +251,7 @@
      */
     @SideEffectFree
     @StaticallyExecutable
-    public @StringVal("") String() {
+    public @StringVal("") @Unique String() {
```

Comment:
Also for this class and StringBuffer below, annotate the other constructors and see whether any other methods should be annotated.

Reply:
I have annotated the remaining constructors as @unique in my lates commit. I also annotated the append function for strings as they were annotated in the stub file.